generatebundlefile/data/prod_artifact_move.yaml

```diff
@@ -4,12 +4,12 @@ kubernetesVersion: "1.22"
 packages:
   - org: aws
     projects:
-      - name: eks-anywhere-packages
-        copyimages: true
-        repository: eks-anywhere-packages
-        registry: public.ecr.aws/l0g8r8j6
-        versions:
-          - name: 0.3.9-eks-a-33
+      # - name: eks-anywhere-packages
```

Comment:
We should probably remove it if we won't be using it.

Reply:
We want to add this back in during the next update so I just kept it there to help people, but can remove it too.

Reply:
removed